Analysis/Tasks/PWGCF/FemtoDream/include/FemtoDream/FemtoDreamContainer.h

```diff
@@ -43,10 +44,13 @@ class FemtoDreamContainer
     mHistogramRegistry = registry;
     std::string femtoObs;
     if (mFemtoObs == femtoDreamContainer::Observable::kstar) {
-      femtoObs = "#it{k}^{*} (GeV/#it{c})";
+      femtoObs = "#it{k*} (GeV/#it{c})";
     }
     mHistogramRegistry->add("relPairDist", ("; " + femtoObs + "; Entries").c_str(), o2::framework::kTH1F, {{5000, 0, 5}});
     mHistogramRegistry->add("relPairkT", "; #it{k}_{T} (GeV/#it{c}); Entries", o2::framework::kTH1F, {{5000, 0, 5}});
+    mHistogramRegistry->add("relPairkstarkT", ("; " + femtoObs + "; #it{k}_{T} (GeV/#it{c})").c_str(), o2::framework::kTH2F, {{5000, 0, 5}, {70, 0, 7}});
```

Comment:
I think we should also make the ranges configurable. 
and the binning of the multiplicity histogram should go along with the mixing